templates.json

```diff
@@ -2,5 +2,7 @@
   {"template_name":  "mnist-training", "workflow_name":  "mnist_workflow_cpu"},
   {"template_name":  "mnist-training", "workflow_name": "mnist_workflow_gpu"},
   {"template_name": "simple-example", "workflow_name": "wf"},
-  {"template_name":  "wine-classification", "workflow_name":  "training_workflow"}
+  {"template_name":  "wine-classification", "workflow_name":  "training_workflow"},
+  {"template_name":  "basic-template-imagespec", "workflow_name":  "wf"},
+  {"template_name":  "hello_world", "workflow_name": "hello_world_wf"}
```

Comment:
not sure why we're adding the `hello-world` example, but the template name is not matching (`hello_world` instead of `hello-world`).

Reply:
I added a`hello-world` example in response to [this feedback](https://github.com/flyteorg/flytesnacks/pull/1266#discussion_r1424487297) on the new onboarding docs, and went into a bit more detail on my changes [here](https://github.com/flyteorg/flytesnacks/pull/1266#issuecomment-1854637502). I'll update the template name, thanks for flagging!